apps/server/src/modules/user-login-migration/service/school-migration.service.spec.ts

```diff
@@ -53,17 +54,23 @@ describe('SchoolMigrationService', () => {
 		const schoolDO: SchoolDO = schoolDOFactory.buildWithId({
 			id: 'schoolId',
 			name: 'schoolName',
-			officialSchoolNumber: '3',
+			officialSchoolNumber: 'officialSchoolNumber',
 			externalId: 'firstExternalId',
 			oauthMigrationFinished: new Date(2023, 2, 27),
 			oauthMigrationStart,
 		});
 
+		const currentUser: ICurrentUser = { userId: 'userId', schoolId: 'schoolId', systemId: 'systemId' } as ICurrentUser;
+
 		const userDO: UserDO = {
 			id: 'userId',
 			schoolId: schoolDO.id as string,
 		} as UserDO;
```

Comment:
A test factory already exists, that can be used.

Reply:
done